static/oas/v1/leagues/leagues.yaml

```diff
@@ -41,4 +62,32 @@ components:
                 #  id: 1
                 #  created_at: "2009-11-10T23:00:00Z"
                 #  updated_at: "2009-11-10T23:00:00Z"
-                #  name: Tripping
\ No newline at end of file
+                #  name: Tripping
+    PostLeaguesRequest:
+      type: object
+      properties:
+        correlation_id:
+          type: string
```

Comment:
I looked at the League model and it is a string for correlation_id, but why? Do you mind asking Skyler if this should change? What is the difference between correlation_id and league_id.

Reply:
I had the same question and definitely wanted to double check this, @Skylude Could you weigh in on this?
What was the idea behind this type being a string rather than a uint or something of that nature?